tabpy-server/tabpy_server/app/app.py

```diff
@@ -320,7 +320,7 @@ def _parse_pwd_file(self):
                             'in password file'.format(row))
                         return False
 
-                    login = row[0]
+                    login = row[0].lower()
```

Comment:
Do we need to document this (that names are converted to lower and not unique based on capitalization)?
And that we will use an English locale to perform the lower() operation...